FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/Readme.md

```diff
@@ -23,6 +23,46 @@ enp0s3:     ethernet interface
 virbr0:     virtual bridge         (to be created)
 virbr0-nic: veth virtual interface (to be created)
 ```
+## Setting up the QEMU Demo
+
+To setup the system for running the QEMU demo, please check these steps.
+```
+1. Setting up the client-server system
+    a. Option-1 : The demo can be run on 2 different systems, one with the client running FreeRTOS system in qemu and another running the echo server.
+    b. Option-2 : The demo could also be run on 1 system, with the QEMU client running on Ubuntu Virtual Machine (VM) and the host OS (Windows/MAC/Linux) running the server.
+
+2. Setting up MAC and IP Address
```

Comment:
Could we specify which definitions would hold each? we always get questions from confused customers that map concepts to actual macros they need to fill up